apps/api/v2/src/app.ts

```diff
@@ -41,16 +41,17 @@ export const bootstrap = (app: NestExpressApplication): NestExpressApplication =
     })
   );
 
+  const { httpAdapter } = app.get(HttpAdapterHost);
+  app.useGlobalFilters(new SentryFilter(httpAdapter));
+
   app.useGlobalFilters(new PrismaExceptionFilter());
 
   if (process.env.SENTRY_DNS) {
```

Comment:
We don't want to start our app if "SENTRY_DNS" is missing, right? So we can remove the if check and have only
```
Sentry.init({
      dsn: getEnv("SENTRY_DNS"),
    });
```

Reply:
I think it should not prevent starting the app, especially if we end up having people self hosting apiv2

Reply:
Ah thats true - agreed!